Incentive program - Deployment script

mtabasco opened the pull request:
This script allows users to fetch, compile, and deploy the 1inch CumulativeMerkleDrop contract from its [github repository](https://github.com/1inch/merkle-distribution/tree/1f8b2a6ed27d1b2d18cf8475e42eece60f41c896) to an Ethereum network.
It provides a CLI-like interface to pass the right config parameters.

andrew-blox reviewed (commented):
1. Why not ts?
2. Maybe its best to use native fetch instead of axios (also a small one just minimizes package.json)
3. You dont need hardhat to deploy?
4. In package.json we have `deploy-ssv-distributor` do you not also maybe want the standard `start` it will still do the same `node ./index.js` but start is a common standard in js package.json to start a script?

mtabasco commented:
> 1. Why not ts?
> 2. Maybe its best to use native fetch instead of axios (also a small one just minimizes package.json)
> 3. You dont need hardhat to deploy?
> 4. In package.json we have `deploy-ssv-distributor` do you not also maybe want the standard `start` it will still do the same `node ./index.js` but start is a common standard in js package.json to start a script?

1. Keep things simple?
2. Changed
3. No, it uses `solc` to compile the contracts on the fly
4. Changed
